aries_cloudagent/anoncreds/routes.py

```diff
@@ -439,6 +446,43 @@ async def rev_list_post(request: web.BaseRequest):
     return web.json_response(result.serialize())
 
 
+@docs(
+    tags=["revocation"],
+    summary="Upload local tails file to server",
+)
+@match_info_schema(RevRegIdMatchInfoSchema())
+@response_schema(RevocationModuleResponseSchema(), description="")
+async def upload_tails_file(request: web.BaseRequest):
+    """
+    Request handler to upload local tails file for revocation registry.
+
+    Args:
+        request: aiohttp request object
+
+    """
+    context: AdminRequestContext = request["context"]
+    profile: AskarProfile = context.profile
+    anoncreds_registry: AnonCredsRegistry = context.inject(AnonCredsRegistry)
+    rev_reg_id = request.match_info["rev_reg_id"]
+    try:
+        issuer = AnonCredsIssuer(profile)
+        rev_reg_def_result = (
+            await anoncreds_registry.get_revocation_registry_definition(
```

Comment:
The check we're performing here should be making sure that we have the rev reg def locally so we can upload the tails file (which should also be present locally). The `get_revocation_registry_definition` method on the anoncreds registry retrieves a rev reg def from a remote registry though. So what I think we want is actually something more like `get_created_revocation_registry_definitions`, similar to the method we have for cred defs.

Reply:
Or rather, I think I was thinking of `match_created_credential_definitions`